packages/dev/parcel-resolver-docs/DocsResolver.js

```diff
@@ -33,10 +33,11 @@ module.exports = new Resolver({
       if (resolved) {
         // HACK: ensure source code is used to build types, not compiled code.
         // Parcel removes the source field from package.json when the code comes from node_modules.
+        // these are full filepaths, so don't check if they start with the pattern, they won't
         if ((
-          /^@(react-spectrum|react-aria|react-stately|internationalized|react-types|spectrum-icons|adobe\/react-spectrum)/g.test(resolved.filePath)
-            || /^(react-aria|react-stately)/g.test(resolved.filePath)
-          ) && resolved.filePath.endsWith('.d.ts') && !resolved.filePath.includes('@react-types')) {
+          /@(react-spectrum|react-aria|react-stately|internationalized|react-types|spectrum-icons|adobe\/react-spectrum)/g.test(resolved.filePath)
+          || /(react-aria|react-stately)/g.test(resolved.filePath)
+        ) && resolved.filePath.endsWith('.d.ts') && !resolved.filePath.includes('@react-types')) {
```

Comment:
nit: you could probably remove `react-types` from the regex and then you wouldn't need this extra `includes` check?